package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "react-native-music-control",
```

Comment:
can you discard these changes or explain why you introduced them? 🙏

Reply:
Hey, not sure how it was on the npm but when pointing directly to github project (what I was doing) the main file is `src/` and not `lib/`. But that was the only reason, if I've misunderstand something this change can be discarded

Reply:
You mean you linked the package locally, right?

in that case you should run `yarn start` or `npm start` within the react-native-music-control project so that files gets trnaspiled ;-) Hope it helps

Reply:
Yes, it was locally, I had not understood, but cool we could just discard those changes. 

I've updated the PR, so just let me know if anything else is needed to fix :)